expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
```

Comment:
@msarahan I haven't tried clang. I have successfully built this package for linux-32, linux-64, and osx-64 as it is now. I just prefer to use as much conda dependecies as possible instead of relying on system-wide packages.